public/surveys.html

```
@@ -0,0 +1,244 @@
+  <div id="site-navbar"></div>
+
+  <main class="flex-1 w-full max-w-[90rem] mx-auto mt-6 px-4 md:px-6 mb-12">
+    <div class="flex flex-wrap gap-4 justify-between items-center mb-6">
+      <h1 class="text-2xl font-bold flex items-center">
+        <i class="fa-solid fa-list-alt text-teal-600 dark:text-teal-400 mr-2" aria-hidden="true"></i>
+        All Surveys
+      </h1>
+      <div class="flex items-center gap-3">
+        <div class="relative">
+          <i class="fas fa-search absolute left-3 top-1/2 -translate-y-1/2 text-gray-400 text-sm"></i>
+          <input id="search" type="text" placeholder="Search surveys…"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)